routes/tools/locales.js

```diff
@@ -0,0 +1,56 @@
+const fetch = require('node-fetch');
+const server = require('../../server').getServer();
+const Boom = require('boom');
+
+var getTranslations = function(tool, lng, next) {
+  const baseUrl = server.settings.app.tools.get(`/${tool}/baseUrl`);
+  fetch(`${baseUrl}/locales/${lng}/translation.json`)
+    .then(response => {
+      if (!response.ok) {
+        throw Boom.create(response.status, response.statusText);
+      }
+      return response.json();
+    })
+    .then(script => {
+      next(null, script);
+    })
+    .catch(err => {
+      if (err.isBoom) {
+        next(err, null);
+      } else {
+        const error = Boom.badRequest();
+        next(error, null);
+      }
+    })
+}
+
+server.method('getTranslations', getTranslations, {
+  cache: {
+    expiresIn: server.settings.app.misc.get('/cache/serverCacheTime'),
+    generateTimeout: 10000
+  }
+});
+
+var route = {
+  method: 'GET',
+  path: '/tools/{tool}/locales/{lng}/translation.json',
+  handler: function(request, reply) {
+    let lng = request.params.lng;    
+    request.server.methods.getTranslations(request.params.tool, lng, (err, result) => {
+      if (err) {
+        return reply(err);
+      }
+      return reply(result);
+    })
+  },
+  config: {
```

Comment:
perfect. if we have a similar issue in the routes of the tools, we can correct it whenever we see it.